lib/controllers/base.js

```diff
@@ -55,18 +56,21 @@ Controller.prototype.route = function() {
 
   // NOTE: is there a better place to do this mapping?
   var pathOptions = endpoint.string;
-  if (app.name === 'restify') {
-    if (self.method === 'delete')
+  if (this.isRestify) {
+    if (self.method === 'delete') {
```

Comment:
because otherwise this is a breaking change for people who depended on this behavior before (me).